frontend/catalyst/third_party/oqd/oqd_database_managers.py

```diff
@@ -0,0 +1,496 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+OQD Device Properties
+~~~~~~~~~~~~~~~~~~~~~
+
+This module defines the classes that represent the properties of an Open Quantum Design (OQD)
+trapped-ion quantum computer device and the methods for loading them from their respective databases
+and configuration files.
+"""
+
```

Comment:
```suggestion

# pylint: disable=line-too-long

```

Reply:
I'd rather not disable this pylint warning at file scope, I think it's a useful warning to have.

The reason we're getting `line-too-long` warnings here is because of two docstrings that are just over the limit with the triple quotation marks. Unfortunately `black` changes lines

```python
"""Long docstring.
"""
```

to

```python
"""Long docstring."""
```

even if the triple quotes make the line longer than the character limit! So even if I fix them, the next time I or anyone else runs `make format` it will revert it back to a single-line docstring and raise the Codefactor warning. Bit of a Catch-22... I propose we just ignore these warnings.